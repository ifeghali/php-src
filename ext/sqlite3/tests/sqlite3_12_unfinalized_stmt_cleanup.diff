016+ object(SQLite3_result)#3 (0) {
017+ }
016- NULL