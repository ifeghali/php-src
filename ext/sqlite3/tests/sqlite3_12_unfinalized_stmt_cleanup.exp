Creating Table
bool(true)
INSERT into table
bool(true)
bool(true)
SELECTING results
array(2) {
  [0]=>
  int(%d)
  [1]=>
  unicode(1) "a"
}
Closing database
bool(true)
Check result was freed
NULL
Done