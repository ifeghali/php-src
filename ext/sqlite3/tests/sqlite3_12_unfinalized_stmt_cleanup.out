Creating Table
bool(true)
INSERT into table
bool(true)
bool(true)
SELECTING results
array(2) {
  [0]=>
  int(1217021333)
  [1]=>
  unicode(1) "a"
}
Closing database
bool(true)
Check result was freed
object(SQLite3_result)#3 (0) {
}
Done